libevmasm/Instruction.h

```diff
@@ -97,6 +104,8 @@ enum class Instruction: uint8_t
 	MSIZE,				///< get the size of active memory
 	GAS,				///< get the amount of available gas
 	JUMPDEST,			///< set a potential jump destination
+	BEGINSUB,           ///< set a potential jumpsub destination -- not part of Instructions.cpp
+	BEGINDATA,          ///< begine the data section -- not part of Instructions.cpp
```

Comment:
You cannot use BEGINDATA and BEGINSUB with libevmasm. I think this is fine to add in a different PR. Note that `BEGINSUB` does not really fit inline assembly because of its immediate arguments.